BurdaMagazinOrg/infinite-module
# INREL-4235 refector wishlist / remove icons from socials template

## [INREL-4235](https://jira.burda.com/browse/INREL-4235) 
- Implement Tipser SDK / 
- Tipser Icon + Wishlist Icon is now in user-navigation container
- New user-navigation styling implementation
- Current tipser-cart-count is visible

## How to test:
- Add to tipser products to cart
- Remove tipser products to cart
- The count should be correct
- Icons on home / home - mobile / article / article - mobile looks good

## Checklist:
